(window.webpackJsonp=window.webpackJsonp||[]).push([[15],{537:function(e,t,n){"use strict";n.r(t);n(30);var o={data:function(){return{svg_path:"/img/projects/macromolecular/svg/",steps:["The sequence is searched through a series of protein related database to find related sequences with known properties related to structure and function.","Dynamic assembly of equation parameters based on availabe information. A series of equations are predetermined                 based on feature analysis. The linear combination of terms are determined from identified similarities in the Protein database.","The initial image is randomly generation based on sequence the prompt and identified information. The less sequence information available the                 further the guess is from the correct solution.","Starting from the initial images random distorations are added to the image. The image improvement are guided by                     an equation. After so many iterations the process is stopped and the current image is saved.","Step 2 and 3 are repeated several thousand times.","The best answer is determined by which image minimizes the equation.","Local features from the neighborhood of the highest rank image are integrated into the model by superimposing                 based on consensus image features and averaging together."],image_super:[{name:"Root Mean Square Deviation",width:"300",image:"rmsd.svg",desc:"Similar to protein sequences, protein structures need to be aligned in order to evaluate the\nsimilarity. Given a superposition, one way is to compare structures is to calculate the root mean\nsquared error between a set of points."},{name:"Expanded RMSD",width:"420",image:"rmsd_expanded.svg",desc:"\nKabsch derived the solution for finding the optimal solution of y onto x as to minimize the RMSD.\nFirst the equation is squared and mean squared deviation is used for simplicity and the equation is\nexpanded."},{name:"Min component",width:"500",image:"maximize.svg",desc:"\nNotice that the first summation term is not dependent on U and can be removed from consideration.\nIn order to minimize the RMSD we need to maximize.\n                "},{name:"Trace",width:"300",image:"min_trace.svg",desc:"\nConverting the equation into vector notation leads to the equation below with the goal of\nmaximizing L.\n                "},{name:"Cyclic Property",width:"300",image:"cyclic.svg",desc:"\nNow using the cyclic property of the transpose the equation can be written as follows where R can\nbe calculated and U is unknown.                \n                "},{name:"SVD",width:"300",image:"svd.svg",desc:"\nR is the correlation matrix and using singular value decomposition it can be rewritten as R = V ∗\nS ∗ W T where V and W are orthogonal matricies and S is a diagonal matrix containing the singular\nvalues. Replacing R with the SVD and using the trace cyclic property\n                "},{name:"orthoganal",width:"300",image:"maximize.svg",desc:"\nThe matrices W T ,U,V are replaced with the matrix T, which is the product of orthogonal matrices,\nwhich leads to"},{name:"identity",width:"300",image:"identity.svg",desc:"\n\nSince T is orthogonal and T ii <= 1, the trace is maximized when T equals the Identity Matrix.\nThe Kabsch algorithm optimal rotation matrices can either be in the right handed coordinate\nsystem or left handed system (determinant of U equals -1). In order to convert a left handed system\nto right handed the last column of U needs to be multiplied by negative one.\n                "},{name:"tmscore",width:"300",image:"tmscore.svg",desc:"\nStructural scoring systems that can properly identify global structure similarity are important.\nThe TMscore was designed to consider the alignment coverage and pairwise distance proximities\nin order to calculate a similarity score.\n                "}],physical_score:{name:"Physical Potential",width:"300",image:"physical_energy.svg",desc:"\nThe initial attempts to determine protein structure and model its dynamics consisted of created\nphysical potentials and incorporating them into a molecular dynamics simulation. The Amber and\nCharm force fields are among the most commonly used. Amber uses the first five and charm adds the second two. The first two terms refer to proper bond\nlengths and angles. The dihedrals refer to the phi and psi angles in the protein backbone. The next\nterms are the Vander Waals and electrostatic potentials. The last terms refer to bond bending.\n                "},info_score:[{name:"Orientation",image:"orientation.svg",desc:"\n For any residue i and j in opposite chains which are in contact, the orientation of the\nunit bisector vectors of i and j can be in three different orientations as defined by their dot product:\nparallel, anti-parallel or perpendicular. e(A i , A j , γ i,j ) is the probability of an amino acid pair to be in the\norientation γ i,j\n                "},{name:"DFIRE",image:"dfire.svg",desc:"Probability of an amino acid being in an local environment of other amino acids"},{name:"Pairwise Amino Preference",image:"amino_preference.svg",desc:"\n Frequence of a pair of amino acids being in close contact. "},{name:"Distance Restraints",image:"dist_map.svg",desc:"Distance restraints obtained from global and local images."}],publications:["Zhang, W., Yang, J., He, B., Walker, S.E., Zhang, H., Govindarajoo, B., Virtanen, J., Xue, Z., Shen, H.B. and Zhang, Y., 2016. Integration of QUARK and I‐TASSER for ab initio protein structure prediction in CASP11. Proteins: Structure, Function, and Bioinformatics, 84, pp.76-86.","Guerler, A., Govindarajoo, B. and Zhang, Y., 2013. Mapping monomeric threading to protein–protein structure prediction. Journal of chemical information and modeling, 53(3), pp.717-725.","Xu, T., Park, S.S., Giaimo, B.D., Hall, D., Ferrante, F., Ho, D.M., Hori, K., Anhezini, L., Ertl, I., Bartkuhn, M. and Zhang, H., 2017. RBPJ/CBF 1 interacts with L3 MBTL 3/MBT 1 to promote repression of Notch signaling via histone demethylase KDM 1A/LSD 1. The EMBO journal, 36(21), pp.3232-3249.","Yang, J., Zhang, W., He, B., Walker, S.E., Zhang, H., Govindarajoo, B., Virtanen, J., Xue, Z., Shen, H.B. and Zhang, Y., 2016. Template‐based protein structure prediction in CASP11 and retrospect of I‐TASSER in the last decade. Proteins: Structure, Function, and Bioinformatics, 84, pp.233-246.","Li, H.D., Menon, R., Govindarajoo, B., Panwar, B., Zhang, Y., Omenn, G.S. and Guan, Y., 2015. Functional networks of highest-connected splice isoforms: from the chromosome 17 human proteome project. Journal of proteome research, 14(9), pp.3484-3491.","Xue, Z., Jang, R., Govindarajoo, B., Huang, Y. and Wang, Y., 2015. Extending protein domain boundary predictors to detect discontinuous domains. PLoS One, 10(10), p.e0141541.","Govindarajoo, B., 2016. Template Based Modeling and Structural Refinement of Protein-Protein Interactions (Doctoral dissertation)."]}}},r=n(35),component=Object(r.a)(o,(function(){var e=this,t=e._self._c;return t("div",[e._m(0),e._v(" "),t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[e._m(1),e._v(" "),t("div",{staticClass:"content"},[t("ol",e._l(e.steps,(function(n,o){return t("li",{key:o,staticClass:"has-text-justified is-size-5"},[e._v(e._s(n))])})),0)])])]),e._v(" "),t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[e._m(2),e._v(" "),t("div",{staticClass:"content"},e._l(e.image_super,(function(n,o){return t("div",{key:o},[t("p",{staticClass:"is-size-5"},[e._v(e._s(n.desc))]),e._v(" "),t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block"},[t("img",{attrs:{src:e.svg_path+n.image,alt:"Centered image"}})])])])})),0)])]),e._v(" "),t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[t("p",{staticClass:"subtitle"},[t("strong",[e._v("Physical Score:")]),e._v(" "+e._s(e.physical_score.desc)+" ")]),e._v(" "),t("div",{staticClass:"content"},[t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block",style:{"min-width":e.physical_score.width+"px"}},[t("img",{attrs:{src:e.svg_path+e.physical_score.image,alt:"Centered image"}})])])])])]),e._v(" "),t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[e._m(3),e._v(" "),t("div",{staticClass:"content"},e._l(e.info_score,(function(n,o){return t("div",{key:o},[t("p",{staticClass:"is-size-5"},[t("strong",[e._v(e._s(n.name)+":")]),e._v("  "+e._s(n.desc))]),e._v(" "),t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block"},[t("img",{attrs:{src:e.svg_path+n.image,alt:"Centered image"}})])])])})),0)])]),e._v(" "),e._m(4),e._v(" "),e._m(5),e._v(" "),e._m(6),e._v(" "),e._m(7),e._v(" "),e._m(8),e._v(" "),t("section",{staticClass:"section"},[t("div",{staticClass:"container"},[e._m(9),e._v(" "),t("div",{staticClass:"content"},[t("ol",e._l(e.publications,(function(n,o){return t("li",{key:o,staticClass:"has-text-justified is-size-5"},[e._v(e._s(n))])})),0)])])])])}),[function(){var e=this,t=e._self._c;return t("section",{staticClass:"section"},[t("div",{staticClass:"container"},[t("h1",{staticClass:"title"},[e._v("Predicting Protein Structure and Interactions")]),e._v(" "),t("p",{staticClass:"subtitle"},[t("strong",[e._v("Prediction Pipeline:")]),e._v(" Starting from a protein sequence how to determine its structure and function?")]),e._v(" "),t("div",{staticClass:"content"},[t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block",staticStyle:{"max-width":"800px"}},[t("img",{attrs:{src:"/img/projects/macromolecular/TacosOverview.png",alt:"Centered image"}})])])])])])},function(){var e=this._self._c;return e("p",{staticClass:"subtitle"},[e("strong",[this._v("Pipeline Steps")])])},function(){var e=this._self._c;return e("p",{staticClass:"subtitle"},[e("strong",[this._v("Image Similarity and Superposition:")])])},function(){var e=this,t=e._self._c;return t("p",{staticClass:"subtitle"},[t("strong",[e._v("Statistical Energy Scores and Information Based Restraints:")]),e._v(" \nInformation from the PDB can improve the modeling and prediction of protein structure. These\nknowledge based simulations incorporate PDB structural information in the form of statistical\npotentials, folding biases, rotamer libraries and homology derived restraints. The statistical\npotentials derived from the PDB are based on the theoretical foundations of the Boltzmann\ndistribution. The Boltzmann distribution is an equation that describes the population of different\nstates and the respective energy of that state. With lower the energies corresponding to the more\npopulated states.\n    ")])},function(){var e=this,t=e._self._c;return t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[t("p",{staticClass:"subtitle"},[t("strong",[e._v("Correlation to Ground Truth:")]),e._v(" To partly assess the quality of our force field, the Pearson’s correlation coefficient was calculated\nbetween the energy and the TM-score of decoys to the structure of the native complex. The\ncorrelation can have variation depending on the quality of the complex and monomeric templates.")]),e._v(" "),t("div",{staticClass:"content"},[t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block",staticStyle:{"max-width":"1000px"}},[t("img",{attrs:{src:"/img/projects/macromolecular/InformationScore.png",alt:"Centered image"}})])])])])])},function(){var e=this,t=e._self._c;return t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[t("p",{staticClass:"subtitle"},[t("strong",[e._v("Predicted Models:")]),e._v(" Near-native models built by TACOS. Plot showing examples of TACOS modeling for both homo- and\nheterodimers. The predicted models are shown in red and slate for chain A and B; the native structure shown as a black line \nis superimposed onto the model structure.")]),e._v(" "),t("div",{staticClass:"content"},[t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block",staticStyle:{"max-width":"1000px"}},[t("img",{attrs:{src:"/img/projects/macromolecular/Generated_models.png",alt:"Centered image"}})])])])])])},function(){var e=this,t=e._self._c;return t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[t("p",{staticClass:"subtitle"},[t("strong",[e._v("Predicting Interaction Networks:")]),e._v(" The entire E. coli genome was modelled by the structure prediction pipelines I-TASSER and\nQUARK. Using SPRING 46,033 dimer complexes were predicted in the E. coli. Using the\npregenerated data from I-TASSER, TACOS can be used to refine and assemble\nthe predicted protein dimeric interactions. The models from I-TASSER and QUARK can be used\nin the initial stage of dimer threading to identify distant homology complexes and improve\nrecognition of conserved binding residues at the interface.")]),e._v(" "),t("div",{staticClass:"content"},[t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-16x9 is-inline-block"},[t("img",{attrs:{src:"/img/projects/macromolecular/InteractionNetwork.png",alt:"Centered image"}})])])])])])},function(){var e=this,t=e._self._c;return t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[t("p",{staticClass:"subtitle"},[t("strong",[e._v("Interaction Network Superposition:")]),e._v(" SPRING is a template-base algorithm for protein-protein complex \n    structure prediction. The pipeline first threads one chain of the protein complex through the PDB library with the binding parters \n    retrieved from the original oligomer entries. The complex models for the query are then deduced from the template binding partner \n    associations through a pre-calculated look-up table. The model of the best orientation is finally selected by the SPRING-score that is a \n    combination of threading Z-score, interface contacts, \n    and TM-align match between monomer-to-dimer templates. The final goal was to build full atom models using TACOS as a final refinement step.")]),e._v(" "),t("div",{staticClass:"content"},[t("div",{staticClass:"has-text-centered"},[t("figure",{staticClass:"image is-1x1 is-inline-block",staticStyle:{"max-width":"400px"}},[t("img",{attrs:{src:"/img/projects/macromolecular/spring_overview.jpg",alt:"Centered image"}})])])])])])},function(){var e=this,t=e._self._c;return t("section",{staticClass:"section",staticStyle:{"padding-bottom":"10px"}},[t("div",{staticClass:"container"},[t("p",{staticClass:"subtitle"},[t("strong",[e._v("Conclusion:")]),e._v(" The general approach to predict macromolecular structures has been significantly starting in 2016. \n    Now deep learning is used to create to energy function to guide the simulations.")]),e._v(" "),t("div",{staticClass:"content"})])])},function(){var e=this._self._c;return e("p",{staticClass:"subtitle"},[e("strong",[this._v("Articles and Publications")])])}],!1,null,null,null);t.default=component.exports}}]);